lib/sprockets/npm.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+require 'json'
+
+module Sprockets
+  module Npm
+    # Internal: Override resolve_alternates to install package.json behavior.
+    #
+    # load_path    - String environment path
+    # logical_path - String path relative to base
+    #
+    # Returns candiate filenames.
+    def resolve_alternates(load_path, logical_path)
+      candidates, deps = super
+
+      # package.json can only be nested one level deep
+      if !logical_path.index('/'.freeze)
```

Comment:
I think you can drop the `.freeze` everywhere, because you already have `frozen_string_literal: true`